tests/ui/rust-2018/removing-extern-crate-malformed-cfg.fixed

```diff
@@ -1,6 +1,7 @@
 //@ edition:2018
 //@ aux-build: remove-extern-crate.rs
 //@ run-rustfix
+//@ rustfix-only-machine-applicable
```

Comment:
This is needed because otherwise it tries to apply the suggestion of the cfg_attr syntax which has holes and therefore fails